src/Containers/RoundTwo/RoundTwo.tsx

```diff
@@ -14,79 +14,74 @@ const RoundTwo:React.FC = () => {
   const [ inputValue, setInputValue ] = useState<string>('');
   const [ responses, setResponses ] = useState<string[]>([]);
   const [ currentStep, setCurrentStep] = useState<number>(0);
-  const [ time, setTime ] = useState<number>(60);//eslint-disable-line
+  const [ time ] = useState<number>(60);
   const dispatch = useDispatch();
   let history = useHistory();
 
```

Comment:
Nice! I didn't know you could stop the array deconstruction with just the value of state